### PR TITLE
Added quotes to fix paths with spaces

### PR DESCRIPTION
**Pull request by markri**
I tried using this module with: 

`store_dir => 'Trusted Root Certification Authorities'`

This will break the "inspect.ps1" script with syntax errors (because of the spaces used in the path) resulting in exit code 1 (even if the certificate is installed). Which results in a "change" on every puppet run.
To overcome this I'm now temporarily using extra quotes in the store_dir parameter

`store_dir => '"Trusted Root Certification Authorities"'`

Note that this does NOT work for fresh deployments. So I use it without the extra quotes to deploy and add the extra quotes afterwards to keep it working

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Thank you for the PR and the explanation. I may be understanding wrong, so feel free to re-open, though it appears that this fix requires the user to change the puppet code after running it. This approach is not idempotent, so I am closing.

**Comment by markri** [comment]
Hi @ghoneycutt This needs to be re-opened. I might not have been clear about my explanation. Let me try again. Following directive fails because it has spaces in the name.

`store_dir => 'Trusted Root Certification Authorities'`

To make this work I created a patch (see the attached commit)

**Comment by ghoneycutt** [comment]
@markri Could you please add a test that shows this works when you have a space in the store_dir. Checkout https://github.com/voxpupuli/puppet-sslcertificate/blob/master/spec/defines/sslcertificate_spec.rb

**Comment by bastelfreak** [comment]
Hi,
based on a discussion in https://groups.io/g/voxpupuli/message/449 we decided to archive this repository. I'm going to close all issues and PRs. If you're interested in maintaining the module, please respond to our mailinglist.
